### ISSUE TITLE
It is impossible to handle Unigram with a screenreader

### ISSUE THREAD

**Issue by Piciok** [UnigramDev/Unigram#564]
Currently, a couple of issues make it impossible for blind users to successfully navigate the Unigram app:
1. The messages in a chat window are not read at all. The only thing exposed to the screenreader is the control and view name, however the message body is not accessible.

2. The chats on the Recent Chats list provide their information in a rather illogical manner, exposing the less relevant information such as the chat Id first and only then the actual chat name

3. Most of the controls in the app such as buttons lack the necessary labels which results in screenreaders not being able to tell what the controls do.

You can learn more on accessibility in universal apps at:

`https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/accessibility.

**Comment by nvdaes** [comment]
Hi, a NVDA user is requesting this can be addressed in the NVDA's add-ons mailing list:

https://nvda-addons.groups.io/g/nvda-addons/message/6317

Thanks for taking care

**Comment by MarcAnt01** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/bd69657bb41d5c9283821a020b3e6db82981f9dc
